build/files.c

```diff
@@ -3107,7 +3107,7 @@ rpmRC processBinaryFiles(rpmSpec spec, rpmBuildPkgFlags pkgFlags,
 			int didInstall, int test)
 {
     Package pkg;
-    rpmRC rc = RPMRC_OK;
+    rpmRC res = RPMRC_OK;
```

Comment:
This seems like gratuitous renaming of variables. Is there a reason you did this?

Reply:
rc stands for "return code", res stands for "result", "return code" is not what this variable means

Reply:
but I can restore `rc` if it is a common variable across the whole code

Reply:
I think it'd be better to restore it to `rc` and if that is something to clean up later, it could be done in a separate commit in another PR.

Reply:
`res` is also used, e.g. in parseForSimple() and in many other palces (`grep -nHr 'res ='`) and usage of `res` there is much more alike the case that we are dicscussing than usage of `rc`

Reply:
res and rc are used pretty much interchangeably throughout, such unrelated gratuitous changes in commits are not okay.

As for the actual change, ACK for the principle of printing all known errors at once but NAK for the patch as long as it spews messages about files that the user is not responsible for.

Reply:
@pmatilai I agree that *.debug files are odd there, may be you know how to fix it rather simply?

Reply:
Maybe we should just filter out `/usr/lib/debug/*` lines from the script?

Reply:
You can't just filter them out as not everybody uses the built-in debuginfo services.

Reply:
Do you mean that not everybode will have ```%__check_files``` defined to that script?

Reply:
No. He means that there are cases where debuginfo packages are manually defined. And this is true. For example, in Fedora, the kernel package, shim, and all MinGW packages define debuginfo generation manually. In that case, the debug files *must* be verified by `check-files`.

Reply:
How reliable is it to see if %_debug_package is %nil to disable filtering *.debug files?